src/Mobs/Monster.cpp

```diff
@@ -696,7 +696,7 @@ void cMonster::KilledBy(TakeDamageInfo & a_TDI)
 			break;
 		}
 	}
-	if ((a_TDI.Attacker != nullptr) && (!IsBaby()))
+	if (a_TDI.Attacker != nullptr)
```

Comment:
Yeah I'm not really confident with this line too. What is exactly the purpose of this line originally ? Because IsBaby returns false in the current version.